packages/editor-codemirror/src/behaviors/completion.ts

```diff
@@ -114,6 +114,8 @@ async function getCompletions(
     return null;
   }
 
+  console.log('getCompletions', completions, [...completions.items])
+
```

Comment:
In vscode, in the extension host, in the developer console: <img width="465" height="315" alt="Image" src="https://github.com/user-attachments/assets/9381affb-1956-47c0-9fce-d1298efb549e" />

In Positron, in the extension host, in the developer console: 
<img width="349" height="639" alt="Image" src="https://github.com/user-attachments/assets/7f0076e1-9c97-4fd1-b612-fcda05740d14" />